src/styles/blocks/heder.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+
+  &__picture {
+    width: 40px;
+    height: 40px;
+    padding: 0;
+  }
+
+  &__navlist-linkstyle {
```

Comment:
The class name `&__navlist-linkstyle` is not following the BEM naming convention. It should be written in lower case and separated by hyphens. So, it should be renamed to `&__navlist-link-style`.